docs/spec.md

```diff
@@ -419,6 +458,38 @@ Unsubscribe from parameter updates. Only supported if the server previously decl
 }
 ```
 
+### Subscribe connection graph updates
+
+Subscribe to [connection graph updates](#connection-graph-update). Only supported if the server previously declared that it has the `connectionGraph` [capability](#server-info).
+
+#### Fields
+
+- `op`: string `"subscribeConnectionGraphUpdate"`
```

Comment:
naming nit: not sure the "Update" part is very useful, we could just call it `(un)subscribeConnectionGraph`